src/main.rs

```diff
@@ -34,15 +40,17 @@ async fn main() {
 
     let scenarios = config::parse_scenarios(config);
 
-    if let Some(_) = options.subcommand_matches("list") {
+    if options.subcommand_matches("list").is_some() {
         list::show_list(&scenarios);
     }
 
     if let Some(run_options) = options.subcommand_matches("run") {
         let scenario_label = run_options.value_of("SET").unwrap();
         let endpoint_url = run_options.value_of("url").unwrap();
+        let api_key = run_options.value_of("API_KEY").unwrap();
         run::run(
             endpoint_url,
+            api_key,
```

Comment:
Since we're using the api key as a default header in http requests to the collector endpoint I am wondering if it would make sense to build an http client with the api key embedded in it :thinking: 

I realized rust provides a `ClientBuilder` that seems to allow this thing
https://docs.rs/reqwest/0.8.4/reqwest/struct.ClientBuilder.html#method.default_headers

No strong opinion, I just see the `api_key` jumping from `run` to `post_fixtures` function and then end up in the http call :smile:

Reply:
Yes that could be an option but I actually don't see the point of passing HTTP client around since it is an implementation detail of post fixture